code/stack.yaml

```diff
@@ -15,7 +15,7 @@
 # resolver:
 #  name: custom-snapshot
 #  location: "./custom-snapshot.yaml"
-resolver: lts-20.26
+resolver: lts-22.31
```

Comment:
the build weirdly broke, and I suspect that this may have something to do with it?

Reply:
I changed this since weeder is not supported on `lts-20.26`

Reply:
I'm not saying that you shouldn't have changed this (clearly, you have no choice), just that this is something to investigate as a potential source of breaking things -- the error message was quite obscure. It might have been because of the typo below too!

Reply:
I also updated the version using the scripts so it should be good now